src/SDKs/_metadata/consumption_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2018-02-03 01:21:07 UTC
+2018-05-18 01:00:05 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      976e0032e80d6fc286298aa3a777b3c6dc2379ca
+Commit:      cf93d01f4fe7c177b24056ff2599a164bd136d68
```

Comment:
Please create a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.